client/src/App.js

```diff
@@ -1,21 +1,29 @@
-import React, { useState } from "react";
-import { BrowserRouter as Router, Route } from "react-router-dom";
-
+import React from "react";
+import { withRouter, Route, Link } from "react-router-dom";
+import { axiosWithAuth } from "./utils/axiosWithAuth";
+import PrivateRoute from "./components/PrivateRoute";
+import { BubblePage } from "./components/BubblePage";
 import Login from "./components/Login";
 import "./styles.scss";
 
 function App() {
+  const loggedIn = axiosWithAuth();
```

Comment:
same issue, the axiosWithAuth returns a function which should always default as true if I am correct